spec/models/article_spec.rb

```diff
@@ -401,35 +405,40 @@ def build_and_validate_article(*args)
 
   describe "#async_score_calc" do
     context "when published" do
-      before { ActiveJob::Base.queue_adapter = :inline }
-
       let(:article) { build(:article) }
 
       it "updates the hotness score" do
-        article.save
-        article.reload
-        expect(article.hotness_score > 0).to eq(true)
+        perform_enqueued_jobs do
```

Comment:
Here and in other tests that require active jobs to be run immediately, we use Rails's own [ActiveJob::TestHelper.perform_enqueued_jobs](https://api.rubyonrails.org/classes/ActiveJob/TestHelper.html#method-i-perform_enqueued_jobs)